### PR TITLE
Non cellular support

### PR DESCRIPTION
**Pull request by ah1053**
1. Remove OS-dependent code to CarrierInfoIntegration.
2. Updated C# Binary.
3. Set Tests to EU Region (UDP not working on Telus Cloudlets)
4. Update DeviceInfoIntegration with IsPingSupported() method.
5. Update ChangeLog.
6. Add overrideCarrierInfo for DeviceInfoIntegration.
7. Add ExampleNonCellular for non-cellular devices.

### CODE REVIEW

**Review by lgarner — APPROVED** [review]
Straightforward. 👍

File: Runtime/Scripts/MobiledgeXIntegrationHelper.cs [hunk 1]
```diff
@@ -324,21 +325,6 @@ public async Task<bool> IsRoaming()
...
-        return new Dictionary<string, string>();
-      }
-      UpdateLocationFromDevice();
-      deviceInfo["NetworkCountryIso"] = await carrierInfoClass.ConvertGPSToISOCountryCode(location.longitude, location.latitude);
-#endif
-      return deviceInfo;
-    }
 
     /// <summary>
     /// Checks whether the default netowrk data path Edge is Enabled on the device or not, Edge requires connections to run over cellular interface.
```
Comment (brucearmstrong) [inline Runtime/Scripts/MobiledgeXIntegrationHelper.cs:330]: Nit: netowrk -> network

File: Tests/Runtime/MobiledgeX_RuntimeTests.cs [hunk 2]
```diff
@@ -41,6 +41,7 @@ public void MobiledgeXEnvironmentSetup()
       {
         Assert.Fail("MobiledgeX Plugins are not loaded in the project, Can't preform tests");
```
Comment (brucearmstrong) [inline Tests/Runtime/MobiledgeX_RuntimeTests.cs:42]: Nit: preform -> perform

File: Runtime/Scripts/MobiledgeXIntegrationHelper.cs (hunk 1 above)
Comment (ah1053) [reply Runtime/Scripts/MobiledgeXIntegrationHelper.cs:330]: Fixed in https://github.com/mobiledgex/edge-cloud-sdk-unity/pull/174/commits/f8c0f09719433b383b86a3b2bcef10f04cfa8c8b

File: Tests/Runtime/MobiledgeX_RuntimeTests.cs (hunk 2 above)
Comment (ah1053) [reply Tests/Runtime/MobiledgeX_RuntimeTests.cs:42]: Fixed in f8c0f09

**Pull request merged**